src/structures/Guild.js

```diff
@@ -585,12 +605,13 @@ class Guild extends Base {
 
   /**
    * Fetches this guild.
+   * @param {boolean} [withCounts=false] Wether or not to fetch the approximate member and presence count
    * @returns {Promise<Guild>}
    */
-  fetch() {
+  fetch(withCounts = false) {
     return this.client.api
       .guilds(this.id)
-      .get()
+      .get({ query: { with_counts: withCounts } })
```

Comment:
To stay consistent to `Client#fetchInvite` this should probably be just `true` and not a parameter.
https://github.com/discordjs/discord.js/blob/88133d0d774e703700fd92ea0a3a63cc78ac4f22/src/client/Client.js#L248-L254